### ISSUE TITLE
bug: Clean install never completes unless you background bootstrap

### ISSUE THREAD

**Issue by matt-bathyscope** [bluerobotics/BlueOS-docker#784]
### Bug description

When running `install.sh` on a clean install of Raspberry Pi OS the installation script never completes unless you background the execution of the bootstrap container.

### Steps to reproduce

1. Install Raspberry Pi OS image on a new micro SD card
2. Run the installation script
3. Script never proceeds beyond the `docker run ...` [here](https://github.com/bluerobotics/BlueOS-docker/blob/master/install/install.sh#L130) because of the infinite loop [here in bootstrap.py](https://github.com/bluerobotics/BlueOS-docker/blob/master/bootstrap/bootstrap/bootstrap.py#L234)

### Primary pain point(s)

You have to manually background the execution of the bootstrap Docker container because of the infinite loop `bootstrap.py` that prints `core is already running, waiting for it to stop...` forever.

### Prerequisites

- [X] I have checked to make sure that a similar request has not already been filed or fixed.

**Comment by matt-bathyscope** [comment]
Output snippet from running `installer.sh`:

```
Going to install companion-docker version master.
Downloading and installing udev rules.
Disabling automatic Link-local configuration in dhcpd.conf.
Downloading bootstrap
master: Pulling from bluerobotics/companion-bootstrap
20cc49690d90: Pull complete 
8dd7fdf9ae3c: Pull complete 
da9874a7d833: Pull complete 
a44ce8d2fc5a: Pull complete 
450d05e80c18: Pull complete 
Digest: sha256:93b9877da3e32b4b88ce3fa6b6b3f5b5938118fbefd77917a0f1d4a618c1cee9
Status: Downloaded newer image for bluerobotics/companion-bootstrap:master
docker.io/bluerobotics/companion-bootstrap:master
WARNING: The requested image's platform (linux/arm/v7) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
unable to read startup.json file ([Errno 2] No such file or directory: '/root/.config/bootstrap/startup.json'), reverting to defaults...
Done
Starting bluerobotics/companion-core
Core started
Done
Could not talk to version chooser for 63.13048458099365: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /version-chooser/v1.0/version/current (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xf703bd10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Reseting startup.json...
450d05e80c18: Pull complete 
Starting bluerobotics/companion-core
Core started
Done
Could not talk to version chooser for 1.5373501777648926: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /version-chooser/v1.0/version/current (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xf6fa9d70>: Failed to establish a new connection: [Errno 111] Connection refused'))
core is already running, waiting for it to stop...
Could not talk to version chooser for 2.571528673171997: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /version-chooser/v1.0/version/current (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xf703bf50>: Failed to establish a new connection: [Errno 111] Connection refused'))
core is already running, waiting for it to stop...
Could not talk to version chooser for 3.6153252124786377: Expecting value: line 1 column 1 (char 0)
core is already running, waiting for it to stop...
Could not talk to version chooser for 4.65287709236145: Expecting value: line 1 column 1 (char 0)
core is already running, waiting for it to stop...
Could not talk to version chooser for 5.676274061203003: Expecting value: line 1 column 1 (char 0)
core is already running, waiting for it to stop...
Could not talk to version chooser for 6.710412979125977: Expecting value: line 1 column 1 (char 0)
core is already running, waiting for it to stop...
core is already running, waiting for it to stop...
core is already running, waiting for it to stop...
core is already running, waiting for it to stop...
core is already running, waiting for it to stop...
```

**Comment by matt-bathyscope** [comment]
One idea here would be to:

1. Launch the container in detached mode (`-d` param)
2. Use a flag file like `/root/.config/bootstrap/.bootstrap_completed` to indicate when the bootstrap process is done, which may take a while to download the various containers
3. Add logic to `install.sh` to wait for the flag file to appear and then proceed with the rest of the script

There are obvious pros and cons to this approach, but ideally the installer would have some way to know when the bootstrap is done without making bootstrap exit. Thoughts?

**Comment by matt-bathyscope** [comment]
This looks like it was resolved in a11058a544c6d4bff2560e2b7b595db03a5d6cc7 by moving from `docker run` to `docker pull` and `docker create`. Closing.
